components/brave_rewards/resources/page/components/pageWallet.tsx

```diff
@@ -745,6 +747,15 @@ class PageWallet extends React.Component<Props, State> {
     )
   }
 
+  getInternalFunds = () => {
+    const { balance } = this.props.rewardsData
+    if (!balance.wallets) {
+      return 0
+    }
+
+    return balance.wallets['anonymous'] || 0 + balance.wallets['blinded'] || 0
```

Comment:
You probably want:

```js
(balance.wallets['anonymous'] || 0) + (balance.wallets['blinded'] || 0)
```

(`+` has higher precedence than `||`).

Reply:
fixed